### PR TITLE
Ported math plugin from dev to es6

### PR DESCRIPTION
**Pull request by avipatel91**
Math plugin evaluates expressions using mathjs. If an invalid expression
is provided, 'invalid input' is returned.

### CODE REVIEW

**Comment by avipatel91** [comment]
It is failing because there is no regex match for messages (e.g. ping) without backslashes in Plugin manager.
[regex location](https://github.com/crisbal/Telegram-Bot-Node/blob/es6/src/PluginManager.js#L184l)

**Comment by CapacitorSet** [comment]
Yes, don't worry about the CI build, if you look at the build history you'll see that it has been failing for some time. I tried fixing that in 5be52071ecdf2ef9b415f2c015d7abe9e1e9aa7e, but it didn't work; if you want to help with that, PRs are certainly welcome.

File: src/plugins/Math.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            result = 'result ' + mathjs.eval(args.join(' '));
+        } catch (e) {
+            result = 'invalid input';
+        }
+
+        reply({
+            type: "text",
+            text: result,
+            options: {
+                parse_mode: "markdown"
```
Comment (CapacitorSet) [inline src/plugins/Math.js]: Why is this line needed?

Comment (avipatel91) [reply src/plugins/Math.js]: To be honest, I do not know. I saw it in the original and copied it over. But now that I look at it, the original had a template literal. Markdown shouldn't really be required for this since I am using a string without any markdowns. I can remove it.

Comment (CapacitorSet) [reply src/plugins/Math.js]: Please remove it, and then the PR will be ready for merging into `es6`.

**Comment by CapacitorSet** [comment]
Thanks!
